pypeit/core/wavecal/autoid.py

```diff
@@ -1772,11 +1774,10 @@ def run_brute(self, min_nlines=10):
 
         # Now that all slits have been inspected, cross match to generate a
         # list of all lines in every slit, and refit all spectra
-        if self._nslit > 1:
+        obad_slits = self.cross_match(good_fit, self._det_weak) if self._ok_mask.size > 1 else np.array([])
+        if obad_slits.size > 1:
```

Comment:
Should this be `>=`? The previous `if self._nslit > 1:` was checking if this was multislit vs longslit, I think, but this new check is looking explicitly for bad slits.